docs/conf.py

```diff
@@ -0,0 +1,58 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+autodoc_mock_imports = ["sqlalchemy"]
+
+master_doc = 'index'
+
+# -- Project information -----------------------------------------------------
+
+project = 'sqlalchemy-ibmi'
+copyright = '2020, Kevin Adler and Naveen Ram'
+author = 'Kevin Adler and Naveen Ram'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.0'
```

Comment:
This should match what's in the setup.cfg and we should add this file to the bumpversion config or we can import the package and inspect its version eg. https://github.com/scrapy/scrapy/blob/2.1/docs/conf.py#L61